lib/internal/process.js

```diff
@@ -65,6 +65,19 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  // 
+  if (global.hasOwnProperty('Intl') &&
+      Intl.hasOwnProperty('v8BreakIterator') &&
+      process.config.variables.icu_small &&
```

Comment:
You still need a `process.config && process.config.variables` feature check here and can you make sure `make lint` passes?